sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py

```diff
@@ -146,6 +156,133 @@ def get_client_access_token(self, *, client_protocol: Optional[str] = "Default",
 
     get_client_access_token.metadata = {"url": "/api/hubs/{hub}/:generateToken"}  # type: ignore
 
+    @distributed_trace
+    def list_connections(
```

Comment:
Why can't you just delegate to a generate method? We really want to avoid copying the whole body of the generated code.